projects/common/src/telemetry/user-telemetry-impl.service.ts

```diff
@@ -71,6 +71,21 @@ export class UserTelemetryImplService extends UserTelemetryService {
       );
   }
 
+  private getDateDiff(timeParamValue: string | undefined): number | undefined {
+    if (timeParamValue !== undefined) {
+      const timeRangeArray = timeParamValue.split('-');
+      const startTime = Number(timeRangeArray[0].trim());
+      const endDate = new Date();
+
+      const diffInTime = endDate.getTime() - startTime;
+      const diffInDays = diffInTime / (1000 * 3600 * 24);
+
+      return Number(diffInDays.toFixed(0));
```

Comment:
```suggestion
      return parseInt(diffInDays);
```

Reply:
The above will not work as parseInt expects `string` as its argument.
Therefore used `toFixed()` to remove the decimal numbers which converts it to string and henceforth used Number() to convert it back to `number`.

Also, can use `parseInt(diffInDays.toFixed(0), 10)`

Reply:
Working for both strings and numbers in the browser console. Is the code throwing error?

<img width="156" alt="Screenshot 2022-12-13 at 6 10 37 PM" src="https://user-images.githubusercontent.com/29686866/207320767-3de79bef-83ab-4ee9-bed8-a4a399657674.png">

Reply:
Just checked, typescript is expecting a string. Nevermind, ignore the comment.